GameData/MOARdV/MAS_ASET/ASET_SignalLamp/MAS_SignalLamp_CommnetCommunicate.cfg

```diff
@@ -36,7 +36,7 @@
 			passiveColor = COLOR_MOARdV_IndicatorLampOff
 			colorName = _EmissiveColor
 			activeColor = COLOR_MOARdV_IndicatorLampGreen
-			variable = fc.Conditioned(fc.CommNetCanCommunicate())
+			variable = fc.Conditioned(fc.CommNetSignalStrength() > 0)
```

Comment:
A couple of comments - first, ASET_SignalLamp files are auto-generated from data in prop.xml, so the entry in that file needs to be updated, or this will be overwritten.  Second, why did this need to change?  The value "Can Communicate" comes from the stock commnet system, so I would prefer adding another signal lamp to track signal strength.

Reply:
1. prop.xml updated
2. My observation is that this lamp doesn't quite work as desired. I will try making a screenshot. With this update it is much more reliable to me.

Reply:
Here's the screenshot: https://i.ibb.co/ZNxd6gN/screenshot-20190101-230241.png

Note that I have a lot of mods installed so this might be only my problem.

Reply:
Okay, I think I see the issue.  Can Communicate only indicates that the vessel is capable of communicating. That is, it has a functional antenna that is capable of providing a commnet connection adequate for vessel control.  CommNetCanScience is equivalent, except that it indicates that the vessel can transmit science.

Neither of them account for loss of signal - I think the better solution is for me to add a signal strength check to both of those functions.  Let's discard this part of the change, and let me push an update to the underlying functions.